libs/langchain/langchain/embeddings/spacy_embeddings.py

```diff
@@ -20,7 +18,9 @@ class SpacyEmbeddings(BaseModel, Embeddings):
             Generates an embedding for a single piece of text.
     """
 
-    nlp: Any  # The Spacy model loaded into memory
+    nlp: Optional[
+        Any
+    ] = "en_core_web_sm"  # The SpaCy model loaded into memory, defaulting to "en_core_web_sm"
```

Comment:
not a big deal but this is the only place where `SpaCy` is written with a capital `C`. the rest of the code including the comments have `Spacy` or `spacy`.